Documentation/getting-started-test-app.md

```diff
@@ -0,0 +1,31 @@
+# Getting started with the test app
+
+## Data Collection mobile property prerequisites
+
+The test app needs to be configured with a Data Collection mobile property with the following extensions before it can be used:
+
+* [Mobile Core](https://github.com/adobe/aepsdk-core-android) (installed by default)
+* Edge Network (this extension)
+* [Consent for Edge Network](https://github.com/adobe/aepsdk-edgeconsent-android)
+* [Identity for Edge Network](https://github.com/adobe/aepsdk-edgeidentity-android)
+* [Assurance](https://github.com/adobe/aepsdk-assurance-android)
+
+See [Configure the Edge Network extension in Data Collection UI](getting-started.md#configure-the-edge-network-extension-in-data-collection-ui) for instructions on setting up a mobile property.
+
+## Run test application
+
+1. In the test app, set your `ENVIRONMENT_FILE_ID` in `TestApplication.java`.
+2. Select the `app` runnable with the desired emulator and run the program.
+
+## Validation with Assurance
+
+Configure a new Assurance session by setting the Base URL to `testapp://main` and launch Assurance in the test app by running the following command in your terminal:
+
+```bash
+$ adb shell am start -W -a  android.intent.action.VIEW -d "testapp://main?adb_validation_sessionid=ADD_YOUR_SESSION_ID_HERE" com.adobe.marketing.mobile.consenttestapp
```

Comment:
I am not sure if this command line does work or not, trying to testing.  But for Edge it should not be com.adobe.marketing.mobile.consenttestapp.

Reply:
Should be this one
adb shell am start -W -a  android.intent.action.VIEW -d "testapp://main?adb_validation_sessionid=33803986-ae78-4679-975e-4bda17ca8b81" com.adobe.marketing.edgetestapp